sdk/apimanagement/apimanagement-custom-widget-scaffolder/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+  },
+  "ignorePatterns": [
+    "src/bin/execute.ts"
```

Comment:
I don't know, you've added this line last week.
I'm guessing it's because it needs to start with `#!/usr/bin/env node` and ESLint want's to enforce copyright there?

Reply:
Oh yeah I remember it is because lint complains about  `#!/usr/bin/env node` being before the license header